wesnoth-utbs.po

```diff
@@ -15787,31 +15822,27 @@ msgstr ""
 
 #. [race]: id=quenoth
 #: data/campaigns/Under_the_Burning_Suns/units/units.cfg:5
-#, fuzzy
-#| msgid "team_name^Quenoth Elves"
 msgid "race^Quenoth Elf"
-msgstr "Elfové z Quenothu"
+msgstr "Quenothský elf"
 
 #. [race]: id=quenoth
 #: data/campaigns/Under_the_Burning_Suns/units/units.cfg:6
-#, fuzzy
-#| msgid "female^Quenoth Druid"
 msgid "race+female^Quenoth Elf"
-msgstr "Quenothská druidka"
+msgstr "Quenothská elfka"
 
 #. [race]: id=quenoth
 #: data/campaigns/Under_the_Burning_Suns/units/units.cfg:7
-#, fuzzy
-#| msgid "team_name^Quenoth Elves"
 msgid "race^Quenoth Elves"
-msgstr "Elfové z Quenothu"
+msgstr "Quenothští elfové"
 
 #. [race]: id=quenoth
 #: data/campaigns/Under_the_Burning_Suns/units/units.cfg:8
 msgid ""
 "Compared to the elves of the Great Continent around the time of Wesnoth, the "
 "Quenoth Elves are far more suited to life in the desert."
 msgstr ""
+"Ve srovnání s elfy Velkého kontinentu z časů wesnothského království jsou "
```

Comment:
v předloze není Kingdom of Wesnoth, ale když už jste ho v překladu použil, tak jsem si vzpomněl, že jsem to před nedávnem dával do pořádku; berte to jako návrh, možná to ani není lepší, zvlášť pokud je (nevím, nehrál jsem) UtBS časově i místně z jiného světa
```suggestion
"Ve srovnání s elfy Velkého kontinentu z časů Wesnothského království jsou "
```

Reply:
Časově ano, místně ne.